powa/static/js/components/DateRangePicker/DateRangePicker.vue

```diff
@@ -202,20 +202,15 @@ function loadRangeShortcut(shortcut) {
   setFromTo(shortcut.from, shortcut.to);
 }
 
-watch(
-  () => from.value,
-  () => {
-    inputFrom.value = from.value;
-    synchronizeFromPicker();
-  }
-);
-watch(
-  () => to.value,
-  () => {
-    inputTo.value = to.value;
-    synchronizeToPicker();
-  }
-);
+watchEffect(() => {
+  inputFrom.value = rawFrom.value;
+  synchronizeFromPicker();
+});
+
+watchEffect(() => {
+  inputTo.value = rawTo.value;
+  synchronizeToPicker();
+});
```

Comment:
Maybe we could use only one `watchEffect`. To be tested.

Reply:
We will synchronise from and to, even if one is changed. Does it matter?

Reply:
Sorry my bad. I forgot that we have 2 different functions for synchronization.